### ISSUE TITLE
Change Copy Project button to Fork icon with Tool Tip

### ISSUE THREAD

**Issue by jgwerner** [3Blades/react-frontend#316]
Change button from `Copy Project` to a button with a `Fork` icon and add a Tool Tip with `Click here to duplicate project`.

User should then have the option to add the project to their User or Team account and confirm project name to complete namespace.

**Comment by raygesualdo** [comment]
We need to be careful about using anything related to "forking" including icons since our copy feature is a duplication, not a fork. No ancestry is maintained when we copy a project. We can add an icon to that button, but it should probably be an icon that indicates duplication.

Currently, there's no option for selecting which account because teams are not yet implemented.

What is the value of letting them change the name?

**Comment by jgwerner** [comment]
@raygesualdo I'm ok with a duplication icon, it does make more sense. The value of letting the user change the name is that they don't necessarily want `Foo-1` or `Foo (Copy)`. The system may return that name as a default option, but the user should have the option to change it if they want to, much like O/S file managers.

**Comment by johngriebel** [comment]
@raygesualdo @jgwerner from what I can remember, most operating systems would just do the copy, and you can rename the file later. This may not be true for windows though, I'm not sure. Additionally, project copying was not (and shouldn't be in my opinion) implemented with the idea of manually renaming the project as part of the copy. Allowing the user to do this essentially makes the "copy" a normal project creation, where the client (react-frontend in this case) must do a _lot_ of additional work to correctly copy all related items such as servers and project files.

**Comment by raygesualdo** [comment]
I agree @johngriebel. If the user wants to change the name after the fact, then they should be able to do that (which they can now). But I don't think that needs to part of the flow when clicking **Copy Project**.

**Comment by jgwerner** [comment]
@raygesualdo @johngriebel I see your point, but this is an important setting since it involves setting up namespaces, etc., unlike just file names. So getting it right the first time is important. A change to the project name later could mean breaking changes for shared assets and collaborators. I think the user should be able to confirm the name, it shouldn't add a lot of complexity to the UX, in my opinion.

If we were to roll out a feature to duplicate a File, then I would agree with creating a name by default and then allowing the user to change the name with a separate step.

**Comment by johngriebel** [comment]
It _would_ add a lot of complexity to either the frontend or backend though. Either the frontend completely scraps project copy and manually `POST`s a project (and accompanying assets), or app-backend has to be reworked to allow this. I'm not sure I understand what changing the project name could break for collaborators or shared assets. It's my intuition that such a rename would happen when the project is initially copied too.

**Comment by jgwerner** [comment]
@johngriebel to clarify, am and saying `namespace` is account + project, even though that is not correct, strictly speaking. And since we are sharing using UUIDs and those don't change, we should be good there. So disregard my previous comment about namespaces. 

On another note, I have to say that it is not a good user experience to have the project name save and have to go to Settings --> Project Name and change the project. Perhaps we should think if a User Interface element that would allow the user to quickly change the name. I have a couple of ideas in mind, perhaps we could talk them over and revisit with a new issue.

**Comment by raygesualdo** [comment]
@jgwerner Is that really going to be that common of a workflow? GitHub keeps there's in the repo Settings page without it causing a UX issue.

**Comment by jgwerner** [comment]
@raygesualdo I don't follow. With GitHub there is only a `Fork`, not a duplication. The `repo` name remains the same because it's context changes. When duplicating projects within the same account or from one account to the other I would suspect that most users would not want to keep `Foo-1` or whatever. What if we had a project named `Titanic` that everyone duplicates to get started? I would doubt that users would like to keep the name `Titanic` if the project is copied from one account to another or keep the name `Titanic-1` if it's copied within the same account.

**Comment by johngriebel** [comment]
This is part of 3blades/3blades#156.
